app/controllers/forms_controller.rb

```diff
@@ -41,14 +41,40 @@ def create
     @form.form_questions = create_form_questions(params[:form][:linked_questions], params[:form][:linked_response_sets])
     respond_to do |format|
       if @form.save
-        format.html { redirect_to @form, notice: save_message(@form) }
         format.json { render :show, status: :created, location: @form }
       else
         format.json { render json: @form.errors, status: :unprocessable_entity }
       end
     end
   end
 
+  # PATCH/PUT /forms/1
+  # PATCH/PUT /forms/1.json
+  def update
+    if @form.status == 'published'
+      render json: { status: 'Published forms cannot be updated.' }, status: :unprocessable_entity
+    else
+      update_successful = nil
+      @form.transaction do
+        # @form.updated_by = current_user
```

Comment:
Is it intended to no longer care about updated_by, or can you remind me if this is something that needs to be added to the db